dispatch/api/views.py

```diff
@@ -73,6 +73,13 @@ class ArticleViewSet(DispatchModelViewSet, DispatchPublishableMixin):
     serializer_class = ArticleSerializer
     lookup_field = 'parent_id'
 
+    def get_serializer_class(self):
```

Comment:
open to suggestions on a better way to do this

Reply:
I'm just worried about cases where an API consumer would want to fetch a list of articles with the full content included